openverse_catalog/dags/common/popularity/sql.py

```diff
@@ -221,13 +235,21 @@ def create_media_popularity_constants_view(
     popularity_constants_idx=IMAGE_POP_CONSTANTS_IDX,
     popularity_metrics=IMAGE_POPULARITY_METRICS_TABLE_NAME,
     popularity_percentile=IMAGE_POPULARITY_PERCENTILE_FUNCTION,
+    pg_timeout: float = timedelta(hours=6).total_seconds(),
 ):
-    postgres = PostgresHook(postgres_conn_id=postgres_conn_id)
+    postgres = PostgresHook(
+        postgres_conn_id=postgres_conn_id, default_statement_timeout=pg_timeout
+    )
     if media_type == AUDIO:
         popularity_constants = AUDIO_POPULARITY_CONSTANTS_VIEW
         popularity_constants_idx = AUDIO_POP_CONSTANTS_IDX
         popularity_metrics = AUDIO_POPULARITY_METRICS_TABLE_NAME
         popularity_percentile = AUDIO_POPULARITY_PERCENTILE_FUNCTION
+    # TO DO: How long does this usually take and how often is it usually run? Is the
+    # intention here to create the materialized view syntax without populating it? If
+    # so, maybe a much shorter timeout would be appropriate, but then I would expect to
+    # see the phrase "WITH NO DATA" here.
```

Comment:
It looks like we renamed the tasks, so we don't get the durations in the grid view for this dag. I'm not sure if you have access to this Rebecca, but here's the task list for these tasks: https://airflow.openverse.engineering/taskinstance/list/?_flt_3_dag_id=recreate_image_popularity_calculation&_flt_3_task_id=create_image_popularity_constants_view

![image](https://user-images.githubusercontent.com/10214785/218855515-1c8b5aca-29cf-457e-a419-bcb7440ec518.png)

Durations ranged from 4-6 hours, but the last time this was run was almost a year ago so I'd expect it to be even higher. Perhaps 12 or 24 hours would make sense?